src/targets/gpu/convolution.cpp

```diff
@@ -121,6 +152,69 @@ shape miopen_convolution::find(context& ctx, const shape& output_shape, std::vec
         MIGRAPHX_THROW("MIOpen Convolution: find convolution failed");
     algo = perf.fwd_algo;
 
+    if(enabled(MIGRAPHX_MIOPEN_FIND_2{}))
```

Comment:
The lines above this, up(and including) to the `workspace` variable should not be ran for find 2.0. Those are the find 1.0 APIs.

Reply:
Done.